### DISCUSSION TITLE
How to find Github runner status using Github actions API.

### DISCUSSION THREAD

**Discussion by pratikkuma** [community/community#64468] [Actions]
### Select Topic Area

General

### Body

HI,

Is there a way to find the **idle** or **active** status of GitHub runner using GitHub actions API. Currently the API fetches the status as **online** rather than showing the status as **active** or **idle**. We can fetch the status from the UI, but I need to fetch it from API. Could someone please help.

curl -L \
  -H "Accept: application/vnd.github+json" \
  -H "Authorization: Bearer <YOUR-TOKEN>" \
  -H "X-GitHub-Api-Version: 2022-11-28" \
  https://api.github.com/orgs/ORG/actions/runners

{
      "id": 12345,
      "name": "test",
      "os": "Linux",
      "status": "online",
      "busy": false,
      "labels": [

**Comment by Rockyread** [comment]
discussions
